docs/userGuide/components/panel.md

```diff
@@ -64,13 +64,32 @@
 
 <tip-box border-left-color="#00B0F0">
   <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
-  <code>primary</code> type<br>
-  <panel header="#### primary type panel" type="primary">
+  <p>Click the Panels to see the expanded style.<p>
```

Comment:
* End with `</p>`
* Actually, not sure whether it is necessary to include this statement though, since `minimized` is already explained earlier, so should be self-explanatory here.